testdata/deps-external-tests.txt

```diff
@@ -1,16 +1,22 @@
 # check that external test deps are added correctly.
-deps -test .
+modcop list -test
 cmp stdout expect-test
-deps .
+
+modcop list
 cmp stdout expect-no-test
 
+# using . should be the same as specifying no args
+modcop list -test .
+cmp stdout expect-test
+
 -- expect-test --
-example.com/a
-fmt
+build example.com/a
```

Comment:
Having `build` as the prefix on every line seems a bit verbose, no? Especially given this is the default behaviour of `modcop list`. Would it be better to have `build` dependencies not annotated at all, and only test dependencies annotated, and even then with a suffix like ` [test]`?

Reply:
I think I'm going to change to just listing one package/module per line, and add a `-testonly` flag, so that the output is directly amenable to other go commands.